packages/flutter_tools/test/general.shard/run_hot_test.dart

```diff
@@ -29,6 +36,92 @@ void main() {
       const NoOpAnalytics(),
     );
   });
+
+  group('signal handling', () {
+    late final _FakeHotCompatibleFlutterDevice flutterDevice;
+
+    setUpAll(() {
+      flutterDevice = _FakeHotCompatibleFlutterDevice(FakeDevice());
```

Comment:
This test group is failing when I run it locally in the default non-randomized order.

`flutterDevice` should be reconstructed for each test case in `setUp`